app/models/user.rb

```diff
@@ -65,9 +68,9 @@ def self.from_omniauth(auth_hash)
     auth = Authorization.from_omniauth(auth_hash)
     auth.user ||= create do |u|
       u.email = auth_hash.info.email
+      u.display_name = auth_hash.info.name
+      u.login = u.display_name.gsub /\s+/, '_'
```

Comment:
Should sub also this remove other non-permitted chars in the login that may creep in?